module/userProjects/helper.js

```diff
@@ -2479,25 +2477,42 @@ function _projectCategories(categories) {
 function _entitiesInformation(entityIds) {
     return new Promise(async (resolve, reject) => {
         try {
+            let bodyData={
+                "id" : entityIds
+            };
+           
+        
+            let entityDetails = await sunbirdService.learnerLocationSearch(bodyData);
+        
+            let entities = entityDetails.data.response;
 
-            let entityData =
-                await coreService.entityDocuments(
-                    entityIds,
-                    ["metaInformation", "entityType", "entityTypeId", "registryDetails"]
-                );
-
-            if (!entityData.success) {
```

Comment:
@VISHNUDAS-tunerlabs check with success value. what will be the value of "success" when API triggering timeout error